# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Xosé, 2018
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-05-27 11:18+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Xosé, 2018\n"
"Language-Team: Galician (https://www.transifex.com/calamares/teams/20061/gl/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: gl\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Instalar paquetes."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "A procesar paquetes  (%(count)d/%(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "A instalar un paquete."
msgstr[1] "A instalar %(num)d paquetes."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "A retirar un paquete."
msgstr[1] "A retirar %(num)d paquetes."

#: src/modules/packages/main.py:725 src/modules/packages/main.py:737
#: src/modules/packages/main.py:765
msgid "Package Manager error"
msgstr ""

#: src/modules/packages/main.py:726
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:738
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:766
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr ""

#: src/modules/networkcfg/main.py:105 src/modules/initcpiocfg/main.py:235
#: src/modules/initcpiocfg/main.py:239 src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:229 src/modules/fstab/main.py:394
#: src/modules/fstab/main.py:400 src/modules/fstab/main.py:428
#: src/modules/initramfscfg/main.py:85 src/modules/initramfscfg/main.py:89
#: src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr ""

#: src/modules/networkcfg/main.py:106 src/modules/initcpiocfg/main.py:240
#: src/modules/openrcdmcryptcfg/main.py:77 src/modules/localecfg/main.py:141
#: src/modules/fstab/main.py:401 src/modules/initramfscfg/main.py:90
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/initcpiocfg/main.py:28
msgid "Configuring mkinitcpio."
msgstr ""

#: src/modules/initcpiocfg/main.py:236 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/mount/main.py:230 src/modules/fstab/main.py:395
#: src/modules/initramfscfg/main.py:86 src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr ""

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr ""

#: src/modules/mkinitfs/main.py:50 src/modules/dracut/main.py:50
msgid "The exit code was {}"
msgstr ""

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr ""

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr ""

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr ""

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr ""

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr ""

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr ""

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr ""

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr ""

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr ""

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr ""

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr ""

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr ""

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr ""

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr ""

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr ""

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd services"
msgstr ""

#: src/modules/services-systemd/main.py:59
#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr ""

#: src/modules/services-systemd/main.py:60
msgid ""
"<code>systemctl {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""

#: src/modules/services-systemd/main.py:63
#: src/modules/services-systemd/main.py:69
msgid "Cannot enable systemd service <code>{name!s}</code>."
msgstr ""

#: src/modules/services-systemd/main.py:65
msgid "Cannot enable systemd target <code>{name!s}</code>."
msgstr ""

#: src/modules/services-systemd/main.py:67
msgid "Cannot enable systemd timer <code>{name!s}</code>."
msgstr ""

#: src/modules/services-systemd/main.py:71
msgid "Cannot disable systemd target <code>{name!s}</code>."
msgstr ""

#: src/modules/services-systemd/main.py:73
msgid "Cannot mask systemd unit <code>{name!s}</code>."
msgstr ""

#: src/modules/services-systemd/main.py:75
msgid ""
"Unknown systemd commands <code>{command!s}</code> and "
"<code>{suffix!s}</code> for unit {name!s}."
msgstr ""

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Tarefa parva de python."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Paso parvo de python {}"

#: src/modules/bootloader/main.py:43
msgid "Install bootloader."
msgstr ""

#: src/modules/bootloader/main.py:614
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""

#: src/modules/bootloader/main.py:782
msgid "Bootloader installation error"
msgstr ""

#: src/modules/bootloader/main.py:783
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr ""

#: src/modules/mount/main.py:42
msgid "Mounting partitions."
msgstr ""

#: src/modules/mount/main.py:88 src/modules/mount/main.py:124
msgid "Internal error mounting zfs datasets"
msgstr ""

#: src/modules/mount/main.py:100
msgid "Failed to import zpool"
msgstr ""

#: src/modules/mount/main.py:116
msgid "Failed to unlock zpool"
msgstr ""

#: src/modules/mount/main.py:133 src/modules/mount/main.py:138
msgid "Failed to set zfs mountpoint"
msgstr ""

#: src/modules/mount/main.py:253
msgid "zfs mounting error"
msgstr ""

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr ""

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr ""

#: src/modules/fstab/main.py:429
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr ""

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr ""

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr ""

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""

#: src/modules/dracut/main.py:27
msgid "Creating initramfs with dracut."
msgstr ""

#: src/modules/dracut/main.py:49
msgid "Failed to run dracut on the target"
msgstr ""

#: src/modules/grubcfg/main.py:28
msgid "Configure GRUB."
msgstr ""

#: src/modules/displaymanager/main.py:524
msgid "Cannot write KDM configuration file"
msgstr "Non é posíbel escribir o ficheiro de configuración de KDM"

#: src/modules/displaymanager/main.py:525
msgid "KDM config file {!s} does not exist"
msgstr "O ficheiro de configuración de KDM {!s} non existe"

#: src/modules/displaymanager/main.py:586
msgid "Cannot write LXDM configuration file"
msgstr "Non é posíbel escribir o ficheiro de configuración de LXDM"

#: src/modules/displaymanager/main.py:587
msgid "LXDM config file {!s} does not exist"
msgstr "O ficheiro de configuración de LXDM {!s} non existe"

#: src/modules/displaymanager/main.py:670
msgid "Cannot write LightDM configuration file"
msgstr "Non é posíbel escribir o ficheiro de configuración de LightDM"

#: src/modules/displaymanager/main.py:671
msgid "LightDM config file {!s} does not exist"
msgstr "O ficheiro de configuración de LightDM {!s} non existe"

#: src/modules/displaymanager/main.py:745
msgid "Cannot configure LightDM"
msgstr "Non é posíbel configurar LightDM"

#: src/modules/displaymanager/main.py:746
msgid "No LightDM greeter installed."
msgstr "Non se instalou o saudador de LightDM."

#: src/modules/displaymanager/main.py:777
msgid "Cannot write SLIM configuration file"
msgstr "Non é posíbel escribir o ficheiro de configuración de SLIM"

#: src/modules/displaymanager/main.py:778
msgid "SLIM config file {!s} does not exist"
msgstr "O ficheiro de configuración de SLIM {!s} non existe"

#: src/modules/displaymanager/main.py:992
msgid "No display managers selected for the displaymanager module."
msgstr ""
"Non hai xestores de pantalla seleccionados para o módulo displaymanager."

#: src/modules/displaymanager/main.py:993
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""

#: src/modules/displaymanager/main.py:1075
msgid "Display manager configuration was incomplete"
msgstr "A configuración do xestor de pantalla foi incompleta"

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr ""

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr ""

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr ""
